http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/leoPubsubMessageSubscriberModels.scala

```diff
@@ -320,6 +320,7 @@ object LeoPubsubMessage {
   }
 
   final case class DeleteAzureRuntimeMessage(runtimeId: Long,
+                                             diskId: DiskId,
```

Comment:
how about we make this optional? I think eventually do want to make it possible to keep disk

Reply:
Hmm... well, currently, WSM doesn't support this as calling delete on the azure runtime deletes all associated child resources. I think we should keep this as non-optional since it wouldn't make much sense to allow keeping the leo record when the WSM record and azure resource are always deleted

Reply:
I don't mean this needs to be cascading down to WSM call...It's just having this as `Option` in the message so that down the line, it'll be easier to support the case where we'd like to keep the disk. For WSM call, we can just do `diskOpt.traverse(x => ???)`